rel-1.0.0/Dockerfile

```diff
@@ -22,8 +22,8 @@ RUN apt-get update \
         zlib1g \
     && rm -rf /var/lib/apt/lists/*
 
-# Install the LKG preview build of the .NET CLI
-RUN wget -q https://dotnetcli.blob.core.windows.net/dotnet/preview/Binaries/Latest/dotnet-dev-debian-x64.latest.tar.gz \
+# Install the LKG rel/1.0.0 build of the .NET CLI
```

Comment:
What do you think of removing this from the comment?  IMO it doesn't add value and is a maintenance issue.